onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -546,11 +546,10 @@ def main():
     if args.input_int32:
         optimizer.change_graph_inputs_to_int32()
 
-    if args.model_type in set(MODEL_TYPES.keys()):
-        if optimizer.is_fully_optimized():
-            logger.info("The model has been fully optimized.")
-        else:
-            logger.info("The model has been optimized.")
+    if "bert" in args.model_type and optimizer.is_fully_optimized():
+        logger.info("The model has been fully optimized.")
+    else:
```

Comment:
Can we change this so that the full list of fused operators prints for all model types? Currently, it is located in `is_fully_optimized` so this change would print the list of fused operators only for `bert` and not other model types.
https://github.com/microsoft/onnxruntime/blob/3b63d85c253c50099c70ba0db6c141b842bc7cda/onnxruntime/python/tools/transformers/onnx_model_bert.py#L464-L523
We can call `get_fused_operator_statistics` before this `if` condition and then pass the `op_count` dictionary to `is_fully_optimized`.